python/transpose_parameters.py

```diff
@@ -84,6 +85,12 @@ def create_markdown_from_df(df, title = ""):
     # Write parameters in a form readable by the model
     df[["Name", "Value"]].set_index("Name").transpose().to_csv(wide_parameter_file, index = False)
     
+    # copy files to the defaul_params folder
+    source_dir = join( "tests", "data" )
+    target_dir = join( "src", "COVID19", "default_params")
+    copy_tree( source_dir, target_dir )
```

Comment:
It is not a problem in that it is just copying extra files. However, it might be make sense to delete the files from default_params and to copy just the ones we know are needed, so there is not a silent failure if we changed the files names at a later date.